src/App.jsx

```diff
@@ -17,13 +23,17 @@ function App() {
 
     return (
         <div className="container">
-            <Header user={user} theme={theme} setTheme={setTheme} />
-            <Tweets tweets={tweets} setTweets={setTweets} user={user} theme={theme}  />
-            <RightSide theme={theme} />
+            <TweetContext.Provider value={{tweets, user, setTweets}}>
+                <ThemeContext.Provider value={{theme, setTheme}}>
+                    <Header theme={theme} setTheme={setTheme}/>
+                    <Tweets theme={theme}  />
+                    <RightSide theme={theme} />
```

Comment:
You can get rid of the prop drilling here now thanks to the context

Reply:
Hi Nathan, thank you for the feedback! 

I don't know what actually happened here - the simplest explanation being that I forgot to commit the last changes I had made. Either way, the start branch now contains what I had stored locally (without the redundant bits of prop drilling), though I'm not sure how I could get those changes to appear on main (unable to merge due to unrelated commit histories)